projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -1066,46 +1166,79 @@ function register_settings() {
 	}
 
-	// Fulfill the settings section callback requirement by returning nothing
-	function carousel_section_callback() {
-		return;
+	/**
+	 * Fulfill the settings section callback requirement by returning nothing.
+	 */
+	public function carousel_section_callback() {
 	}
 
-	function test_1or0_option( $value, $default_to_1 = true ) {
-		if ( true == $default_to_1 ) {
-			// Binary false (===) of $value means it has not yet been set, in which case we do want to default sites to 1
+	/**
+	 * Tests if a value is set
+	 *
+	 * @param mixed $value The value passed into this function with which to test.
+	 * @param bool  $default_to_1 Default is true.
+	 *
+	 * @return bool
+	 */
+	public function test_1or0_option( $value, $default_to_1 = true ) {
+		if ( true === $default_to_1 ) {
+			// Binary false (===) of $value means it has not yet been set, in which case we do want to default sites to 1.
 			if ( false === $value ) {
 				$value = 1;
 			}
 		}
-		return ( 1 == $value ) ? 1 : 0;
+		return ( 1 === $value ) ? 1 : 0;
 	}
 
-	function sanitize_1or0_option( $value ) {
-		return ( 1 == $value ) ? 1 : 0;
+	/**
+	 * Ensures the value returned is in the correct format.
+	 *
+	 * @see test_1or0_option()
+	 * @param mixed $value The value returned from the test_1or0_option function.
+	 *
+	 * @return int
+	 */
+	public function sanitize_1or0_option( $value ) {
+		return ( 1 === $value ) ? 1 : 0;
 	}
 
-	function settings_checkbox( $name, $label_text, $extra_text = '', $default_to_checked = true ) {
+	/**
+	 * Adds settings checkbox
+	 *
+	 * @param string $name - For name attribute.
+	 * @param string $label_text - For label attribute.
+	 * @param string $extra_text - Additional checkbox description text. Defaults to empty.
+	 * @param bool   $default_to_checked - If the checkbox is checked. Default is true.
+	 *
+	 * @return string
+	 */
+	public function settings_checkbox( $name, $label_text, $extra_text = '', $default_to_checked = true ) {
 		if ( empty( $name ) ) {
 			return;
 		}
 		$option = $this->test_1or0_option( get_option( $name ), $default_to_checked );
 		echo '<fieldset>';
 		echo '<input type="checkbox" name="' . esc_attr( $name ) . '" id="' . esc_attr( $name ) . '" value="1" ';
 		checked( '1', $option );
-		echo '/> <label for="' . esc_attr( $name ) . '">' . $label_text . '</label>';
+		echo '/> <label for="' . esc_attr( $name ) . '">' . esc_html( $label_text ) . '</label>';
 		if ( ! empty( $extra_text ) ) {
-			echo '<p class="description">' . $extra_text . '</p>';
+			echo '<p class="description">' . esc_textarea( $extra_text ) . '</p>';
 		}
 		echo '</fieldset>';
 	}
 
-	function settings_select( $name, $values, $extra_text = '' ) {
+	/**
+	 * Adds settings options
+	 *
+	 * @param string $name - For name attribute.
+	 * @param string $values - For the different option values.
+	 * @param string $extra_text - Additional option section description text. Defaults to empty.
+	 *
+	 * @return string
```

Comment:
Much like the above, this directly outputs rather than returning anything.
```suggestion
	 * Output a selection list options
	 *
	 * @param string $name - For name attribute.
	 * @param string $values - For the different option values.
	 * @param string $extra_text - Additional option section description text. Defaults to empty.
```

Reply:
Sounds good!